cross/libgpg-error/Makefile

```diff
@@ -11,6 +11,13 @@ HOMEPAGE = http://www.gnupg.org/related_software/libgpg-error/
 COMMENT  = Libgpg-error is a small library that defines common error values for all GnuPG components
 LICENSE  = LGPL
 
+PRE_CONFIGURE_TARGET = preConfigureTarget
 GNU_CONFIGURE = 1
 
 include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: preConfigureTarget
+preConfigureTarget:
+	cp $(WORK_DIR)/$(PKG_DIR)/src/syscfg/lock-obj-pub.arm-unknown-linux-gnueabi.h $(WORK_DIR)/$(PKG_DIR)/src/syscfg/lock-obj-pub.linux-gnueabi.h
```

Comment:
Not sure about these two lines (L21-22). Seems to me this won't work for all arches, will it? I'll probably take a better look at it later.